test_journal.py

```diff
@@ -170,28 +170,19 @@ def test_post_to_add_view(app):
     assert entry_data['title'] in actual
 
 
-def test_get_to_add_view(app):
-    entry_data = {
-        'title': 'test',
-        'text': 'text'
-    }
-    response = app.get('/add', params=entry_data, status='404 Not Found')
-    assert response.status_code == 404
-
-
```

Comment:
rather than delete this test, it would be good to verify that using GET to access the add view loads an empty form. Similarly, with the edit view, that it loads a form with the expected values from the entry to be edited.

It'd also be nice to have tests that prove that an anonymous user can't do this.